InfoScreen.c

```diff
@@ -42,14 +42,19 @@ InfoScreen* InfoScreen_done(InfoScreen* this) {
 void InfoScreen_drawTitled(InfoScreen* this, const char* fmt, ...) {
    va_list ap;
    va_start(ap, fmt);
+
+   char* title = xMalloc(COLS + 1);
+   int len = vsnprintf(title, COLS + 1, fmt, ap);
+   if (len > COLS) memset(&title[COLS - 3], '.', 3);
+
```

Comment:
Could we try to move this allocation into the lifetime of the window?

Reply:
I think this is not needed:
`InfoScreen_drawTitled` is called in the `_draw` callbacks of the Screen classes.
The `_draw` callback is called (besides the initial one) on window-resize, F5 key and Crtl-L key.
These events are rather rare, so caching seems not worth it and on resize we would to re-compute anyway.

Reply:
Okay. Sounds reasonable.